R/axis.R

```diff
@@ -27,7 +27,7 @@
 #'            col="blue", pch=18, legend.name="Points", xlab="Index") %>%
 #'    axis(side=c(3,4), labels=FALSE)
```

Comment:
Look here at how it use to look (ie...I want ticks with no labels on side 3 and 4)....I would think both strategies above should work (but I've ignored >2 sides so far...so that 1st example didn't use to work).

https://github.com/USGS-R/gsplot/issues/14

 (ignore the inset for now, the main plot has that sort of workflow as above)